docs/configuration/examples.md

```diff
@@ -107,40 +107,47 @@ create-archive:
 
 </details>
 
-### Custom changelog generation
+### Custom archive creation for release syncing
+
+If you need to generate local archive(s) during release syncing, you can utilise e.g. `pre-sync` action
+to place the commands necessary for the creation. You also have to include the archive(s) in the list of files to
+be moved to the dist-git repo so that it is then handled by Packit from there.
+
+
 <details>
-  <summary>Using changelog entry from a file</summary>
+  <summary>Creating archive using tito and syncing it to dist-git repo</summary>
 
 ```yaml
-changelog-entry:
-  - cat .changelog_entry
+actions:
+  pre-sync:
+    - tito build -o . --tgz
+
+files_to_sync:
+  - src:
+    - "<my-package>-*.tar.gz"
+    dest: .
 ```
 
 </details>
 
+
+### Custom changelog generation
 <details>
-  <summary>Using `git log` output as a changelog entry </summary>
+  <summary>Using changelog entry from a file</summary>
```

Comment:
Just because the title below has this structure and I think it is good.

```suggestion
  <summary>Using a file as a changelog entry</summary>
```

Reply:
I agree with Maja, but I would suggest this:
```suggestion
  <summary>Using file content as a changelog entry</summary>
```

Reply:
thanks for the suggestions!